resources/js/components/Compare.vue

```diff
@@ -29,8 +33,17 @@ export default {
             return compare.value?.item_count;
         },
 
-        attributes: () => {
-            return compare.value?.attributes
+        attributes: function () {
+            return this.onlyShowDifferent ? this.differentAttributes : compare.value?.attributes
+        },
+
+        differentAttributes: function () {
+            let firstItem = this.items[0];
```

Comment:
Will this component exclusively be executed when there are items? If not we might want a failsafe here to return an empty list

Reply:
If there are no items, there will be an empty state on the page and this won't get executed